docs/docs/tools.md

```diff
@@ -15,13 +15,15 @@ Airbyte เป็นเครื่องมือโอเพ่นซอร์
 
 https://airflow.apache.org/
 
-Airflow หรือ Apache Airflow คือโอเพ่นซอร์สแพลตฟอร์มที่เราสามารถที่จะตรวจสอบ ทำตารางเวลา และเฝ้าสังเกต กระแสงาน หรือ Workflow ขั้นตอนการประมวลผลได้
+Airflow หรือ Apache Airflow คือโอเพ่นซอร์สแพลตฟอร์มประเภท orchestration ที่เราสามารถที่จะตรวจสอบ ทำตารางเวลา และเฝ้าสังเกต กระแสงาน หรือ Workflow ขั้นตอนการประมวลผลได้
```

Comment:
I think the term you're looking for is `task orchestrator`

Reply:
ขอบคุณมากเลยครับ แก้แล้วครับ